docs/userguide/configuration.rst

```diff
@@ -1143,6 +1154,103 @@ Example configuration
     cassandra_write_consistency = 'ONE'
     cassandra_entry_ttl = 86400
 
+.. _conf-s3-result-backend:
+
+S3 backend settings
+-------------------
+
+.. note::
+
+    This s3 backend driver requires :pypi:`boto3`.
+
+    To install, use :command:`boto3`:
```

Comment:
Perhaps it is easier and more intuitive to just use the backend name instead of the library name directly. So that would be to install:

```
pip install celery[s3]
```

Reply:
Seems legit :)